classes/asserter/resolver.php

```diff
@@ -50,13 +52,28 @@ public function resolve($asserter)
 		{
 			$class = $this->checkClass($asserter);
 		}
-		else foreach ($this->namespaces as $namespace)
+		else
 		{
-			$class = $this->checkClass($namespace . '\\' . $asserter);
+			if (false === test::isValidIdentifier($asserter))
```

Comment:
Adding a hard dependency on the `test` class here does not seem clean.

I would rather duplicate the `isValidIdentifier` method in the `resolver` class for now and see how we can extract this in a dedicated class later.